agent_deploy/kubernetes/sysdig-agent-daemonset-v1.yaml

```diff
@@ -69,8 +69,14 @@ spec:
             memory: 1024Mi
         readinessProbe:
           exec:
-            command: [ "test", "-e", "/opt/draios/logs/running" ]
+            command: [ "test", "-e", "/opt/draios/logs/startup" ]
           initialDelaySeconds: 10
+          periodSeconds: 5
+        startupProbe:
+          exec:
+            command: [ "test", "-e", "/opt/draios/logs/startup"]
+          failureThreshold: 18
+          periodSeconds: 10
```

Comment:
Can you comment on what these do?

failureThreshold: 18
periodSeconds: 10

Reply:
startup probe failure does not take action before 18 consecutive failure, 10 secs between attempts. In other words, the agent has 180 sec to startup before the probe restarts the container

Reply:
Note that the agent has a max connection backoff timeout of 300 seconds. With this change, the agent will be restarted before that max backoff timeout is hit. I'd recommend making this probe's overall deadline to be at least 300.

Reply:
Good Point. Thank you @balinaveen